src/io/image_aug_default.cc

```diff
@@ -218,6 +257,80 @@ class DefaultImageAugmenter : public ImageAugmenter {
       res = src;
     }
 
+    if (param_.random_resized_crop) {
```

Comment:
Add checks for the mutual exclusion relationship between random_resized_crop and (resize, crop, ctc)

Reply:
Do you mean unittest, or sanity checks in C++?

Reply:
sanity checks